src/main.cpp

```diff
@@ -37,6 +37,8 @@
 #include <Simulator.h>
 #include <Temp.h>
 #include <gpio.h>
+#include <IOExt.h>
```

Comment:
What is it used for?

Reply:
Shouldn't we at least call `init_ioext` from the main file?

Reply:
of course, sorry